localstack/services/s3/s3_listener.py

```diff
@@ -881,8 +882,29 @@ def bucket_exists(bucket_name):
     return True, 200
 
 
+def strip_chunk_signatures(body, headers):
+    if headers.get("x-amz-content-sha256", None) != "STREAMING-AWS4-HMAC-SHA256-PAYLOAD":
+        return body
+    content_length = int(headers["x-amz-decoded-content-length"])
```

Comment:
nit: Can we always expect this header to be present - perhaps better (to avoid `KeyError`):
```
     content_length = int(headers.get("x-amz-decoded-content-length", 0))
```
Alternatively, we could also raise an explicit error message, in case this header is missing.

Reply:
yep, I was thinking about this too, I'll update this.